content/en/blog/2022/exponential-histograms/index.md

```diff
@@ -0,0 +1,308 @@
+percentage of the samples.
+
+I ran the simulation, and exported the histograms via to compare the explicit
+bucket histogram to the exponential bucket histogram. The next two charts show
+the results. The exponential bucket histogram has significantly more detail,
+which simply isn't available with the more limited buckets of the explicit
+bucket histogram.
+
+> **Note:** These visualizations are from the New Relic platform, which I used
+> because they employ me, and it's the easiest way for me to visualize
+> histograms. Every platform will have its own mechanism for storing and
+> retrieving histograms, which typically perform some lossy translation of
+> buckets into a normalized storage format—New Relic is no exception.
+> Additionally, the visualization doesn't clearly delineate buckets, which
+> causes adjacent buckets with the same count to appear as a single bucket.
+
+Here's the millisecond scale exponential bucket histogram:
+
+![millisecond scale exponential bucket histogram](millisecond-scale-exponential-bucket-histogram.webp)
+
+Here's the millisecond scale explicit bucket histogram:
+
+![millisecond scale explicit bucket histogram](millisecond-scale-explicit-bucket-histogram.webp)
+
+This demonstration is fairly generous to the explicit bucket histogram because I
+choose to report values in an optimum range for the default buckets (for
+example, 0 to 1000). The next two examples show what happens when the same
+values are recorded in nanosecond precision instead of milliseconds (all values
+are multiplied by 10<sup>6</sup>). This is where the no-configuration
+autoscaling nature of exponential bucket histograms really shines. Left with the
+default explicit bucket boundaries, all the samples fall into a single bucket
+with the explicit bucket histogram. The exponential variety loses some
+definition compared to the millisecond version in the previous example, but you
+can still see the response time bands.
+
+Here's the nanosecond scale exponential bucket histogram:
+
+![nanosecond scale exponential bucket histogram](nanosecond-scale-exponential-bucket-histogram.webp)
+
+Here's the nanosecond scale explicit bucket histogram:
+
+![nanosecond scale explicit bucket histogram](nanosecond-scale-explicit-bucket-histogram.webp)
+
+## Next steps
+
+Exponential bucket histograms are a powerful new tool for metrics. While
+implementations are still in progress at the time of publishing this post,
+you'll definitely want to enable them when you're using OpenTelemetry metrics.
+
+If you're using [opentelemetry-java][] (and eventually other languages), the
+easiest way to enable exponential bucket histograms is by setting the
+[environment variable][] with this command:
+
+```shell
+export OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION=exponential_bucket_histogram
+```
+
+For instructions on enabling in other languages, check the relevant
+documentation on [opentelemetry.io][] or [github.com/open-telemetry][].
```

Comment:
```suggestion
To enable exponential bucket histograms in other languages, see the language-specific documentation under [Instrumentation][].
```

If you agree with this change then you'll need to change the link definition on line 307 below from

    [opentelemetry.io]: /docs/instrumentation

to

    [Instrumentation]: /docs/instrumentation